source/code/source/playfab/PlayFabCallRequestContainer.cpp

```diff
@@ -22,8 +22,16 @@ namespace PlayFab
         errorWrapper.UrlPath = url;
 
         Json::Value request;
-        Json::Reader reader;
-        bool parsingSuccessful = reader.parse(requestBody, request);
+        std::string errs;
+        Json::CharReaderBuilder builder;
+        Json::CharReader* reader = builder.newCharReader();
+
+        const char* reqBod = requestBody.c_str();
+        size_t reqBodLength = requestBody.length();
+
+        bool parsingSuccessful = reader->parse(reqBod, reqBod + reqBodLength, &request, &errs);
+
+        delete reader;
 
```

Comment:
instead of explicit delete I suggest you manage the reader pointer with a smart pointer (i.e. unique_ptr)